templates/pool.conf.j2

```diff
@@ -0,0 +1,209 @@
+# {{ ansible_managed }}
+;;;;;;;;;;;;;;;;;;;;;;
+; Pool Configuration ;
+;;;;;;;;;;;;;;;;;;;;;;
+
+; The address on which to accept FastCGI requests.
+; Valid syntaxes are: 'ip.add.re.ss:port', 'port', '/path/to/unix/socket'.
+; This option is mandatory for each pool.
+listen = {{ listen | mandatory }}
+
+; Set listen(2) backlog.
+; A value of '-1' means unlimited.
+; Default value: -1.
+listen.backlog = {{ pool.listen.backlog | default(listen.backlog) }}
+
+; List of IPv4 addresses of FastCGI clients which are allowed to connect.
+; Equivalent to the FCGI_WEB_SERVER_ADDRS environment variable
+; in the original PHP FastCGI (5.2.2+).
+; Makes sense only with a tcp listening socket.
+; Each address must be separated by a comma.
+; If this value is left blank, connections will be accepted from any ip address.
+; Default value: any. IPv6 addresses are allowed since PHP 5.5.20 and 5.6.4.
+listen.allowed_clients = {{ pool.listen.allowed_clients | default(allowed_clients) }}
```

Comment:
default(allowed_clients) won't work. If listen is replaced and does not define allowed_clients, the attribute is missing